### PR TITLE
Create "Getting data into LSDB" documentation

### LINKED ISSUE #225: Add loading data section to the documention
Add content to https://lsdb.readthedocs.io/en/latest/tutorials/loading_data.html

Describe various options on how to load data, argument, and possible choices. Connect to hipscat-import concepts and doc pages.

### PR DESCRIPTION
**Pull request by camposandro**
Revamp the import catalogs notebook and move its content to the "Getting data into LSDB" section of the documentation. Closes #225.

### CODE REVIEW

**Comment by camposandro** [comment]
Closing this PR according to changes in https://github.com/astronomy-commons/lsdb/pull/406.

**Comment by camposandro** [comment]
@smcguire-cmu I noticed the "Getting data" section was in fact still empty. I added the contents we agreed for this section (according to https://github.com/astronomy-commons/lsdb/issues/398#issuecomment-2292192876).

File: docs/tutorials/getting_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,177 @@
+{
```
Comment (delucchi-cmu) [inline docs/tutorials/getting_data.ipynb:1]: <p>1) margin caches - add pointer to margin cache topic notebook</p><p>2) pre-select - also say that you can see the schema later</p><p> </p><p>or switch to a flow like:</p><p> </p><p>gaia_dr3 = read_hipscat(..., WITHOUT the columns arg)</p><p>gaia_dr3.schema</p><p> </p><p>Let's just use the columns we need instead:</p><p> </p><p>gaia_dr3 = read_hipscat(..., WITH the columns arg)</p><p>gaia_dr3.schema</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (smcguire-cmu) [inline docs/tutorials/getting_data.ipynb:1]: <p>What does it look like without the title? I think it's a little confusing what the argument is here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (smcguire-cmu) [inline docs/tutorials/getting_data.ipynb:1]: <p>I'm not sure we want to advertise this method. It reads the point_map from disk, which only works when the catalog has just been loaded. Could also be a bit confusing with the difference between hipscat and lsdb packages, and that we have to use the hc_structure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (camposandro) [reply docs/tutorials/getting_data.ipynb:1]: It just says "moll", it doesn't look good. We pass the plot's title as an argument in other tutorials as well.

Comment (camposandro) [reply docs/tutorials/getting_data.ipynb:1]: Thanks. I just changed the flow, it feels closer to what users would experience!

**Review by delucchi-cmu — APPROVED** [review]

**Pull request merged**